config/buildspec_build_wheel.yml

```diff
@@ -0,0 +1,38 @@
+# Build Spec for AWS CodeBuild CI to build smdebug pip wheels
+
+version: 0.2
+
+phases:
+  install:
+    commands:
+        - su && apt-get update
+        - apt-get install sudo -qq -o=Dpkg::Use-Pty=0
+        - sudo apt-get update -qq -o=Dpkg::Use-Pty=0
+        - sudo apt-get install unzip -qq -o=Dpkg::Use-Pty=0
+        - cd $CODEBUILD_SRC_DIR && chmod +x config/protoc_downloader.sh && ./config/protoc_downloader.sh
+        - pip install -U pip
+        - pip install -q pytest wheel pyYaml pytest-html pre-commit
+        - pip uninstall -y boto3 && pip uninstall -y aiobotocore && pip uninstall -y botocore
+
+  build:
+    commands:
+      - cd $CODEBUILD_SRC_DIR  && if [ -z "$CORE_COMMIT" ]; then export CORE_COMMIT=$(git log -1 --pretty=%h); fi
```

Comment:
Let's put env variables in ENV section at top . Example
env:
  variables:
    ## comment, 
    CORE_COMMIT: latest
   RULES_COMMIT

Reply:
these are already declared as env variables in the codebuild project. I can also include it as env variables in the buildspec before the phases.